### ISSUE TITLE
Error detected while processing CursorMoved Autocommands

### ISSUE THREAD

**Issue by zeitchef** [kyazdani42/nvim-tree.lua#437]
I've just noticed that when navigating the tree with "j" or "k", I'm suddenly getting the following error:

```
Error detected while processing CursorMoved Autocommands for "NvimTree":
E5108: Error executing lua ...m/site/pack/packer/start/nvim-tree.lua/lua/nvim-tree.lua:204: Column value outside range
```

This seems only to apply to a particular directory. When I try to open this directory with "o" - everything in the tree is suddenly listed doubled.

As this is a new setup, I'm currently trying to bisect the issue by testing for conflicting plugins, but have had no success so far. Any ideas?

**Comment by kyazdani42** [comment]
Hello, are you sure you are using the latest master ? I've introduced a commit (319620513) which contained the bug which doubled every entry in the tree, but i then added another commit (714dee57e5) which fixed the issue.

**Comment by kyazdani42** [comment]
could you provide me with a minimal config from which i could reproduce this issue please ?
and also if you can tell me which OS you use ?

**Comment by zeitchef** [comment]
Of course, I'm on macOS 11.4. I've been able to isolate the issue as occurring when used with the 'auto-session' plugin.

**Comment by zeitchef** [comment]
This minimal config produces the error:
```
return require('packer').startup(function()
  use 'wbthomason/packer.nvim'
  use { 'kyazdani42/nvim-tree.lua', branch = 'master' }
  use 'rmagatti/auto-session'
end)
```

**Comment by kyazdani42** [comment]
thanks for the minimal config, i can now reproduce perfectly !
I will fix this soon

**Comment by kyazdani42** [comment]
hum cannot reproduce the issue with this config anymore, does this still issue exists on your side ?

**Comment by zeitchef** [comment]
I think there might have been an update in the auto-session plugin - I also haven't seen this issue in a few days. I think it can be closed.

**Comment by wookayin** [comment]
This is still happening to me; I am not using auto-session but it probably conflicts with others. Can we please have this reopened?

```
E5108: Error executing lua .....vim/plugged/nvim-tree.lua/lua/nvim-tree.lua:229: Column value outside range
```

**Comment by kyazdani42** [comment]
haven't seen this issue ever, so it might be related to something else in your configuration. Are you sure all your plugins are up to date ?

**Comment by hiql** [comment]
I got the same error  in zemode :(

 plugin: 'folke/zen-mode.nvim'

**Comment by kyazdani42** [comment]
do you have the `nvim_tree_hijack_cursor` enabled ?

**Comment by hiql** [comment]
> do you have the `nvim_tree_hijack_cursor` enabled ?

I did not set this option.
OK, let me try again, wait a minute...

**Comment by kyazdani42** [comment]
it is enabled by default, i think i should disable this

**Comment by hiql** [comment]
> it is enabled by default, i think i should disable this

Yes, you are right, It's working again~~~thx...
